adulbrich/osu-fsf
# Web event stats + overview design

The large changes in this pull request include

/events/[EventID]/Overview
/events/[EventID]/Stats

To test this
- Run npm install
- navigate to src-web
- npm run dev
- check the overview design and see if anything is wrong (click on an event card on the home page)
- go to the stats tab of the navbar for "Only Steps" and check if the graph shows some data (should be really low numbers because there isn't a lot of test data)
